apps/fluent-tester/src/TestComponents/CornerRadius/CornerRadiusTest.tsx

```diff
@@ -30,8 +30,8 @@ const CornerRadiusTestComponent: React.FunctionComponent<CornerRadiusTestCompone
 
   const innerCircleIndicatorProps = React.useMemo(
     () => ({
-      height: props.cornerRadius * 2,
-      width: props.cornerRadius * 2,
+      height: props.cornerRadius === globalTokens.corner.radiusCircular ? 60 : props.cornerRadius * 2,
+      width: props.cornerRadius === globalTokens.corner.radiusCircular ? 60 : props.cornerRadius * 2,
```

Comment:
Could we do like a `Math.min()` function?

Reply:
Sure, is there a reason you prefer that

Reply:
IMO looks cleaner, and is more extensible for future token updates. Also is more general logic that works, rather than checking for a specific value.